contracts/AssetPool.sol

```diff
@@ -123,10 +139,25 @@ contract AssetPool is Ownable {
             covAmount > 0,
             "Underwriter token amount must be greater than 0"
         );
+        // Ensure withdrawal not initiated or withdrawal initiation prevention
+        // timeout has not elapsed
+        //slither-disable-next-line incorrect-equality
+        require(
+            withdrawalInitiatedTimestamp[msg.sender] == 0 ||
+                withdrawalInitiatedTimestamp[msg.sender].add(
+                    withdrawalInitiationPreventionTimeout
+                ) >
```

Comment:
If we do it this way, underwriters will be able to free-ride for some part of the graceful withdrawal period:
- Eve initiates the withdrawal for 100 000 KEEP.
- Eve waits for the withdrawal delay, earns rewards for that period.
- Eve waits for some part of the graceful withdrawal period.
- Now, Eve can withdraw immediately and even front-run coverage claims. She can also decide not to withdraw by initiating the withdrawal for 1e-18 KEEP if the risk lowered as a result of some market movement. 

It is being discussed [here](https://github.com/keep-network/coverage-pools/issues/60#issuecomment-848695084) and I would rather let her increase her withdrawal but only during the withdrawal delay period.

Reply:
That makes sense.